cmd/backup-manager/app/import/manager.go

```diff
@@ -51,24 +47,6 @@ func NewRestoreManager(
 	}
 }
 
-func (rm *RestoreManager) setOptions(restore *v1alpha1.Restore) {
```

Comment:
We also have to check the nil of `backup.spec.from` in the `cmd/backup-manager/app/export/manager.go`.